sc-memory/cpp/sc_memory.hpp

```diff
@@ -186,10 +186,11 @@ class ScMemoryContext
   SC_DEPRECATED(0.4.0, "Use ScMemoryContext::HelperFindBySystemIdtf(std::string const & sysIdtf) instead.")
   _SC_EXTERN bool HelperFindBySystemIdtf(std::string const & sysIdtf, ScAddr & outAddr);
   _SC_EXTERN ScAddr HelperFindBySystemIdtf(std::string const & sysIdtf);
-  _SC_EXTERN bool HelperGenTemplate(ScTemplate const & templ, ScTemplateGenResult & result, ScTemplateGenParams const & params = ScTemplateGenParams::Empty, ScTemplateResultCode * resultCode = nullptr);
+  _SC_EXTERN bool HelperGenTemplate(ScTemplate const & templ, ScTemplateGenResult & result, ScTemplateParams const & params = ScTemplateParams::Empty, ScTemplateResultCode * resultCode = nullptr);
   _SC_EXTERN bool HelperSearchTemplate(ScTemplate const & templ, ScTemplateSearchResult & result);
   _SC_EXTERN bool HelperSearchTemplateInStruct(ScTemplate const & templ, ScAddr const & scStruct, ScTemplateSearchResult & result);
   _SC_EXTERN bool HelperBuildTemplate(ScTemplate & templ, ScAddr const & templAddr);
+  _SC_EXTERN bool HelperBuildTemplate(ScTemplate & templ, ScAddr const & templAddr, const ScTemplateParams & params);
```

Comment:
```suggestion
  _SC_EXTERN bool HelperBuildTemplate(ScTemplate & templ, ScAddr const & templAddr, const ScTemplateParams & params = ScTemplateParams());
```
One function instead of two:
```cpp
_SC_EXTERN bool HelperBuildTemplate(ScTemplate & templ, ScAddr const & templAddr);
_SC_EXTERN bool HelperBuildTemplate(ScTemplate & templ, ScAddr const & templAddr, const ScTemplateParams & params);
```

Reply:
done